README.md

```diff
@@ -2068,6 +2081,108 @@ aem {
 }
 ```
 
+Custom configuration for the particular OSGi component can be created by [OsgiFramework](src/main/kotlin/com/cognifide/gradle/aem/common/instance/service/osgi/OsgiFramework.kt) as well:
```

Comment:
nice that you are explaning almost all possible cases, but maybe concentrate on 2 good examples with additional info about factory cases only explained - without example.

I am trying to keep readme.md not too long